partiql-planner/src/test/kotlin/org/partiql/planner/PlannerErrorReportingTests.kt

```diff
@@ -264,28 +247,30 @@ internal class PlannerErrorReportingTests {
                 "not_a_function(1)",
                 false,
                 assertOnProblemCount(0, 1),
-                StaticType.MISSING
+                StaticType.ANY
             ),
             TestCase(
                 "not_a_function(1)",
                 true,
                 assertOnProblemCount(0, 1),
-                StaticType.MISSING
+                StaticType.ANY
             ),
 
             // 1 + not_a_function(1)
             //  The continuation will return all numeric type
+            // TODO: Should the warning count be 1? Does it matter if it is zero?
             TestCase(
                 "1 + not_a_function(1)",
                 false,
-                assertOnProblemCount(1, 1),
-                StaticType.MISSING,
+                assertOnProblemCount(0, 1),
+                StaticType.unionOf(StaticType.INT4, StaticType.INT8, StaticType.INT, StaticType.FLOAT, StaticType.DECIMAL),
             ),
+            // TODO: Should the warning count be 1? Does it matter if it is zero?
```

Comment:
Not sure why the tests are repeated. Seems like one can be deleted? Or maybe `signal` should be `true` for this one?